tests/unit/drm_plugin_test.sh

```diff
@@ -396,7 +396,7 @@ function test_convert_module_info()
   assertEquals "$LINENO" "$expected" "$output"
 
   output=$(convert_module_info "LOAD" "")
-  assertEquals "$LINENO" "$?" "22"
+  assertEquals "$LINENO" "22" "$?"
```

Comment:
You can delete the double-quotes around `22`, as pure integers don't need to be surrounded by any quotes.